tests/test_23_pyopenms_api_compat.py

```
@@ -0,0 +1,125 @@
+    return sp
+
+
+def _experiment_with_two_analyzers():
+    exp = oms.MSExperiment()
+    for i in range(6):
+        exp.addSpectrum(_ms1(10.0 + i, [100.0 + i, 200.0 + i, 300.0], [10.0, 20.0, 5.0],
+                             faims_cv=-45.0 if i % 2 == 0 else -55.0))
+        exp.addSpectrum(_ms2(10.5 + i, 500.0 + i, 1000.0, 2, oms.Precursor.ActivationMethod.HCD))
+    # Attach two mass analyzers to the instrument.
+    inst = exp.getInstrument()
+    a0 = oms.MassAnalyzer(); a0.setType(oms.MassAnalyzer.AnalyzerType.ORBITRAP); a0.setResolution(60000.0)
```

Ruff (0.15.21)

[error] Multiple statements on one line (semicolon)

(E702)